cmd/cri-containerd/options/options.go

```diff
@@ -163,6 +176,8 @@ func (c *CRIContainerdOptions) AddFlags(fs *pflag.FlagSet) {
 		defaults.ProfilingAddress, "Profiling address for web interface host:port/debug/pprof/.")
 	fs.BoolVar(&c.SkipImageFSUUID, "skip-imagefs-uuid",
 		defaults.SkipImageFSUUID, "Skip retrieval of imagefs uuid. When turned on, kubelet will not be able to get imagefs capacity or perform imagefs disk eviction.")
+	fs.Var(&c.Registry, "registry",
```

Comment:
I think `registry` means the general registry config.

For this flag, I prefer `registry-mirrors` like Docker, and accept `,` separated url list. Because it only configures mirrors as is described in the description.

In the future:
1) If we still keep cri-containerd flags (won't be the case I guess), we can gradually deprecate `registry-mirrors` support and add new `registry` flag. In that way, we can define new `registry` flag however we want, and don't need to consider backward compatibility.
2) If we switch to pure config file based, the flags will be removed. In that way, the flag name doesn't need to be forward looking, just keep it as what it is for for now, which is `registry-mirrors`. :)

What do you think?

I'm fine with keeping the `Registry struct`, because we may extend it in the future.